Eugene/MovieSearch/my-app/src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./App.css";
+import MainPage from "./components/MainPage/MainPage";
+import MoviePage from "./components/MoviePage/MoviePage";
+import NotFoundPage from "./components/NotFoundPage/NotFoundPage";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { Provider } from "react-redux";
+import store from "./redux/store";
+
+class App extends React.Component {
```

Comment:
as you don't use state or lifecycle methods, better to rewrite to a functional component